### DISCUSSION TITLE
Terrestrial Radiation API documentation

### DISCUSSION THREAD

**Discussion by maximilian-nowottnick-SuS** [open-meteo/open-meteo#514] [Q&A]
Hello community, I was browsing through the very helpful API documentation at https://open-meteo.com/en/docs#hourly=terrestrial_radiation and saw that _terrestrial_radiation_ is not explained in detail, so I was wondering what exactly is returned here.

Compared to ERA5 hourly names (https://cds.climate.copernicus.eu/cdsapp#!/dataset/reanalysis-era5-single-levels?tab=overview), I could imagine it to be 

- Mean surface downward long-wave radiation flux
- Mean surface net long-wave radiation flux
- Mean top net long-wave radiation flux
- Surface net thermal radiation
- Surface thermal radiation downwards
- Top net thermal radiation

And I wanted to ask if anyone is able to clarify what exactly is returned.

**Comment by patrick-zippenfenig** [comment]
Hi,

`terrestrial_radiation` is extraterrestrial solar irradiance without any clouds and atmosphere on a horizontal pane. IIRC it is the same as top of the atmosphere solar radiation (horizontal pane). It is calculated directly by the Open-Meteo API and is not downloaded from CDS ERA5. It is based on the NREL solar position model,  solar inclination calculation multiplied by the solar constant. More information here: https://github.com/open-meteo/open-meteo/blob/001fcc0a983e6599b2c8405dae0d78daa2def6de/Sources/App/Helper/Zensun.swift#L512
